superset-frontend/src/views/CRUD/data/dataset/AddDatasetModal.tsx

```diff
@@ -107,7 +109,7 @@ const DatasetModal: FunctionComponent<DatasetModalProps> = ({
       show={show}
       title={
         <>
-          <StyledIcon name="warning-solid" />
+          <StyledIcon iconColor={theme.colors.grayscale.base} />
```

Comment:
Hey @pkdotson same comment as the other PR. What's making defining the color specifically for the icon necessary here? The icon should get the color from the container

Reply:
oh I see. I'll redo this!

Reply:
@pkdotson is there follow-up work to do on this?

Reply:
@pkdotson bump ^

Reply:
@rusackas @pkdotson see my comment here https://github.com/apache/superset/pull/14463#discussion_r635963140. I believe this falls under the same scenario.

Reply:
Going to keep it color prop for not until we can come up with default color for icon